src/core/identifiers/identifiers.ts

```diff
@@ -133,3 +129,20 @@ export const parseDSNPAnnouncementURI = (
   const [userId, contentHash] = announcementUri.replace("dsnp://", "").split("/");
   return { userId, contentHash };
 };
+
+export const serializeToHex = (
```

Comment:
I realize this is partially tested through all the other functions in this file, but can we test `serializeToHex()` explicitly?

Reply:
Ah forgot about docs on it as well. It was originally not exported. I'll fix. Or I think I will move to a different module and add tests and docs.